src/nft/components/details/detailsV2/DataPageTraits.tsx

```diff
@@ -1,26 +1,26 @@
 import { Trans } from '@lingui/macro'
+import { GenieAsset } from 'nft/types'
 
-import { Tab, TabbedComponent } from './TabbedComponent'
+import { Tab, TabbedComponent, TabTitleWithBubble } from './TabbedComponent'
 
 const TraitsContent = () => {
-  return <div>Traits Content</div>
+  return <div style={{ height: '492px' }}>Traits Content</div>
 }
 
 enum TraitTabsKeys {
   Traits = 'traits',
 }
 
-const TraitTabs: Map<string, Tab> = new Map([
-  [
-    TraitTabsKeys.Traits,
-    {
-      title: <Trans>Traits</Trans>,
-      key: TraitTabsKeys.Traits,
-      content: <TraitsContent />,
-    },
-  ],
-])
-
-export const DataPageTraits = () => {
-  return <TabbedComponent tabs={TraitTabs} style={{ height: '528px' }} />
+export const DataPageTraits = ({ asset }: { asset: GenieAsset }) => {
+  const TraitTabs: Map<string, Tab> = new Map([
```

Comment:
I think this should be in a `useMemo`

Reply:
There is no complex operations happening in this initialization, just map construction. I believe useMemo in this case would be less efficient